bin/test-workflow

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+cd "$(dirname "$0")/.." || ( echo "cannot cd into parent dir" && exit 1 )
+
+function announce() {
+    echo "
+    ===================
+    ${1}
+    ===================
+    "
+}
+
+# Install Conjur in our cluster, and load policies
+mkdir -p temp &2> /dev/null
+pushd temp
+    git clone https://github.com/cyberark/conjur-oss-helm-chart.git &2> /dev/null
+
+    pushd conjur-oss-helm-chart/examples/kubernetes-in-docker
+        announce "Installing Conjur-OSS"
+        helm uninstall conjur-oss
+        ./start
+    popd
+popd
+
+namespace="app-test"
```

Comment:
Do we need to create the "app-test" Namespace here? Isn't it already created with the example scripts that were just run?